scripts/pi-hole/php/header.php

```diff
@@ -452,7 +452,7 @@ function pidofFTL()
                 <!-- Whitelist -->
                 <li<?php if($scriptname === "whitelist"){ ?> class="active"<?php } ?>>
                     <a href="list.php?l=white">
-                        <i class="fa fa-check-circle-o "></i> <span>Whitelist</span>
+                        <i class="fa fa-pencil-square-o"></i> <span>Whitelist</span>
```

Comment:
This was purposely set to `fa-check-circle-o` from `fa-pencil-square-o`, so it should not be changed back.